site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Push an NGINX Plus image to a private registry"
```

Comment:
You're adding this file back in but it's been deleted. May need to fix your rebsae

Reply:
Yes